src/main/java/com/maxmind/geoip2/record/Traits.java

```diff
@@ -45,195 +45,17 @@
     private final Double staticIpScore;
 
     public Traits() {
-        this(null, null, null, null, false, false, null, null, null);
-    }
-
-    public Traits(String ipAddress) {
-        this(null, null, null, ipAddress, false, false, null, null, null);
+        this(null, null, null, null,
+                null, false, false, false, false,
+                false, false, false, false, false, null,
+                null, null, null, null, null, null, null);
     }
 
     public Traits(String ipAddress, Network network) {
```

Comment:
What's using this one? Actually, what uses the no argument variants too?

Reply:
Hmm, I am not entirely sure. I think Jackson used to use the no argument ones, but we switched how we use it. Maybe they can be removed too. I'll try.

Reply:
This and the no-argument ones are used from the constructors for the model classes as we guarantee that things like `.getTraits()` will return an object even if there was no traits data in the response.